packages/cli-kit/src/output.ts

```diff
@@ -228,6 +234,17 @@ export function content(strings: TemplateStringsArray, ...keys: (ContentToken |
         case ContentTokenType.Green:
           output += colors.green(stringifyMessage(enumToken.value))
           break
+        case ContentTokenType.LinesDiff:
+          ;(enumToken.value as unknown as Change[]).forEach((part) => {
```

Comment:
I think `prettier` got confused here.

Reply:
Yeah, but to be honest this casting shouldn't be necessary, but given the way `...keys` is typed it's hard to make this generic. Maybe we should disable prettier for this line.